workers/unity/Assets/Playground/Scripts/MonoBehaviours/WorkerFlagTest.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Playground
 {
-    [WorkerType(WorkerUtils.UnityGameLogic)]
+    [WorkerType(WorkerUtils.UnityGameLogic, "FakeWorkerType")]
```

Comment:
```suggestion
    [WorkerType(WorkerUtils.UnityGameLogic)]
```

Reply:
I thought this was a decent addition for a test? And it shows off the inspector nicely

Reply:
Maybe use one of the existing workers instead?